process/request_resources.md

```diff
@@ -0,0 +1,60 @@
+# Requesting Resources
+
+All OpenSSF projects receive support and resources from the Foundation. The type of resources and amount vary by project stage; refer to the [project lifecycles](process/project_lifecycle.md) for details. 
+
+- Common requests
+    - [Blogs](#communications-and-content)
+    - [Event space](#contribuor-meet-ups-at-OpenSSF-events)
+    - [Infrastructure](#infrastructure)
+    - [Security audits](#security-audits-and-improvements)
+    - [Code of Conduct support](#code-of-conduct-support)
+
+## Requests through the OpenSSF Marketing Committee
+
+### Communications and content
+
+To post on the OpenSSF blog, projects should contact the OpenSSF Marketing Committee with a blog draft two weeks before the requested publication date. The Committee does their best to accommodate requested publication dates, but projects should plan to be flexible during high-traffic times (e.g. events). Posts and timing is at the editorial discretion of the Marketing Committee; posts must follow the Code of Conduct, be relevant to the OpenSSF mission, and not promote vendor or commercial products. 
+
+To receive space in an OpenSSF newsletter, projects should contact the Marketing Committee a minimum of two weeks before the newsletter publication date. Projects are responsible for tracking due dates and submitting materials.
+Technical content should be reviewed by the TAC, and potentially other WG's or Committees as appropriate. This includes, but is not limited to: white papers, technical briefings, policy recommendations, etc. This does not include project-specific announcements, such as those relating to project milestone releases. 
+
+### Contributor meet ups at OpenSSF events
+
+Projects that would like contributor meet up space at OpenSSF events should contact the OpenSSF Marketing Committee with a request six months before the event date, or as soon as the event is announced if it is less than six months. Late requests may be accommodated but are not guaranteed. The events team handles room reservations, attendee registration if necessary, and connecting the project leads to event and facility support staff. Projects are responsible for organizing and running their own contributor meet up.
```

Comment:
Right now the MC isn't set up to handle inbound requests for event support - bring those to operations@openssf.org, and we will turn to the MC or other parts of the OpenSSF community as appropriate.

I am eager to find a way to more systematically support regional meetups and volunteer-run events as CNCF does, possibly using the platform they use, Bevy.

Reply:
Addressed in PR#117